packages/svelte/src/link.ts

```diff
@@ -54,22 +54,31 @@ function link(
   let baseParams: VisitOptions
   let visitParams: VisitOptions
 
-  const regularEvents: ActionEventHandlers = {
+  const regularEvents = {
     click: (event) => {
       if (shouldIntercept(event)) {
         event.preventDefault()
         router.visit(href, visitParams)
       }
     },
-  }
-
-  const prefetchHoverEvents: ActionEventHandlers = {
-    mouseenter: () => (hoverTimeout = setTimeout(() => prefetch(), 75)),
-    mouseleave: () => clearTimeout(hoverTimeout),
+    mouseenter: () => {
+      hoverTimeout = setTimeout(() => prefetch(), 75)
+    },
+    mouseleave: () => {
+      clearTimeout(hoverTimeout)
+    },
+  } satisfies ActionEventHandlers
+
+  const prefetchHoverEvents = {
+    mouseenter: regularEvents.mouseenter,
+    mouseleave: regularEvents.mouseleave,
+    touchstart: regularEvents.mouseenter,
+    focus: regularEvents.mouseenter,
+    blur: regularEvents.mouseleave,
     click: regularEvents.click,
-  }
+  } satisfies ActionEventHandlers
```

Comment:
I replaced the type assignment with the [`satisfies` operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator) to make autocomplete smarter. Now typescript knows which properties from `ActionEventHandlers` are defined and which aren't while still validating the types are correct.